data/src/main/java/com/teamagam/gimelgimel/data/response/entity/ImageResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.teamagam.gimelgimel.data.response.entity;
+
+import com.teamagam.gimelgimel.data.response.entity.contents.ImageMetadataData;
+import com.teamagam.gimelgimel.data.response.entity.visitor.IResponseVisitor;
+
+/**
+ * Created on 5/18/2016.
+ * Image-Type class for {@link GGResponse}'s inner content
+ */
+public class ImageResponse extends GGResponse<ImageMetadataData> {
```

Comment:
Same thing about this one and all the actual message.
(naming + outdated comments).